db/migrations/20220109175923_user.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public."User"
+(
+    "user_ID" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "login" text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
What does the `COLLATE pg_catalog."default"` do?

Reply:
default sorting parameters, i guess we dont need custom one right now

Reply:
Do we need sorting parameter?